Src/java/engine/src/test/java/org/opencds/cqf/cql/engine/execution/RuntimeTests.java

```diff
@@ -42,13 +42,13 @@ void intervalOfQuantityWithDifferentUOM() {
     @Test
     void tupleToString() {
         Tuple t = new Tuple();
-        assertThat(t.toString(), is("Tuple { : }"));
+        assertThat(t.toString(), is("Tuple {\n\n}"));
```

Comment:
nitpick: Isn't it nicer for completely empty Tuples to be a one-liner rather than be spread across several lines?

`Tuple {}`
vs
```
Tuple {

}
```

Reply:
Agreed and applied